### ISSUE TITLE
[Question] How to deal with async funciton?

### ISSUE THREAD

**Issue by Himself65** [darukjs/daruk#75]
in the official website

the only one example is:

```ts
// src/glues/mysql/index.ts
import { Daruk } from "daruk";
import mysql from "./connect";

export default function(daruk: Daruk) {
  return mysql;
}
```

but in real production, my code is:

```ts
export default async function (daruk: Daruk) {
  const mongodb = await mongoose.connect(dbUrl, {
    useCreateIndex: true,
    useNewUrlParser: true
  })
  const db = mongodb.connection
  db.on('connected', () => {
    debug(`Mongoose connection open to ${dbUrl}.`)
  })
  db.on('error', (err) => {
    debug('Mongoose connection error: ' + err)
    throw err
  })
  db.on('disconnected', () => {
    debug('Mongoose connection disconnected')
  })
  return db
}
```

but I read the source code, I think this will run error

**Comment by xiaojue** [comment]
https://github.com/darukjs/daruk/blob/developer/example/02-comments/glues/connection.ts#L7

https://github.com/darukjs/daruk/blob/developer/example/02-comments/services/CommentsModel/index.ts#L9

**Comment by Himself65** [comment]
thanks

―― by Himself65 Outlook<https://aka.ms/qtex0l>
________________________________
发件人: xiaojue <notifications@github.com>
发送时间: Sunday, May 19, 2019 9:50:03 AM
收件人: darukjs/daruk
抄送: 扩散性百万甜面包; Author
主题: Re: [darukjs/daruk] [Question] How to deal with async funciton? (#75)


Closed #75<https://github.com/darukjs/daruk/issues/75>.

―
